addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -26,8 +26,10 @@ def _action_send_mail(self, auto_commit=False):
         return super()._action_send_mail(auto_commit=auto_commit)
 
     def _prepare_mail_values(self, res_ids):
-        """ When being in mass mailing mode, add 'mailing.trace' values directly
-        in the o2m field of mail.mail. """
+        """ In mass mailing only:
```

Comment:
While we are it, could we convert those to comments ? That way smart IDEs don't override the original docstring :) 

(aka: # in mass maililng only: ... blabla)